### PR TITLE
fix(Prejoin): Allow changing 'Enable pre meeting screen' option while prejoin screen visible

### PR DESCRIPTION
**Pull request by vp8x8**

### CODE REVIEW

File: react/features/base/settings/reducer.js [hunk 1]
```diff
@@ -53,7 +53,9 @@ const filterSubtree = {};
 
 // start with the default state
 Object.keys(DEFAULT_STATE).forEach(key => {
-    filterSubtree[key] = true;
+    if (typeof filterSubtree[key] === 'undefined') {
```
Comment (quitrk) [inline react/features/base/settings/reducer.js:56]: doesn't this break something for the other keys that were initialized with `false`?

Comment (vp8x8) [reply react/features/base/settings/reducer.js:56]: Actually this piece of code was not doing what it was promising. There was no way of setting a default value to false (or true for that matter). This does not change the fact that these settings will be overwritten by the local storage values.

Comment (vp8x8) [reply react/features/base/settings/reducer.js:56]: Right, this will mess the default state.

Comment (vp8x8) [reply react/features/base/settings/reducer.js]: Fixed

**Review by quitrk — APPROVED** [review]

**Pull request merged**